src/Symfony/Component/Notifier/Bridge/SmsSluzba/composer.json

```diff
@@ -0,0 +1,31 @@
+{
+    "name": "symfony/smssluzba-notifier",
+    "type": "symfony-notifier-bridge",
+    "description": "Symfony sms-sluzba.cz Notifier Bridge",
+    "keywords": ["sms", "sms-sluzba.cz", "notifier"],
+    "homepage": "https://symfony.com",
+    "license": "MIT",
+    "authors": [
+        {
+            "name": "Dennis Fridrich",
+            "email": "fridrich.dennis@gmail.com"
+        },
+        {
+            "name": "Symfony Community",
+            "homepage": "https://symfony.com/contributors"
+        }
+    ],
+    "require": {
+        "php": ">=8.2",
+        "symfony/http-client": "^7.1",
+        "symfony/notifier": "^7.1",
+        "symfony/serializer": "^7.1"
```

Comment:
We can allow lower version here as well I think

Reply:
Unfortunately yes. They have actually 3 endpoints you can use, but if you want to get SMS ID back you have to use this one with password in URL. I do not like it as well, this is really old but still much used service. I hope they will update some day.

By the way they return HTTP 200 to all responses even errored ones.